scripts/form-editor-support.js

```diff
@@ -64,6 +65,12 @@ export function handleWizardNavigation(wizardEl, navigateTo) {
   navigateToMenuItem.classList.add('wizard-menu-active-item');
 }
 
+export function handleAccordionNavigationInEditor(accordionEl, navigateTo) {
```

Comment:
Why is this method exported if not used outside this module?

Reply:
Removed.